docker/docker-compose.ci.yml

```diff
@@ -24,8 +24,6 @@ services:
     image: unlock
     user: node
     working_dir: /home/unlock
-    depends_on:
```

Comment:
Even though we are now polling via the configuration, I think we might probably need to ensure that at least one of the instances of ganache is ready/available here

Reply:
Oh, so technically I think the `unlock` image itself does not need `ganache-integration` integration to run (it could theoretically be running in other contexts), but the `integration-test` does (and has it!)